centinel/views.py

```diff
@@ -15,9 +17,9 @@
 from werkzeug import secure_filename
 
 
-from centinel.models import Client, Role
-import config
+from centinel.as_info import ASInfo
```

Comment:
I really don't like underscores in python filenames because I know that python has some weird importing rules (e.g. it is a syntax error to try to import a filename that contains hyphens). I wasn't sure where to put this in the actual file, so I'm just putting it here. Also, our naming convention so far has been CamelCase for filenames.